src/Carbon/Laravel/ServiceProvider.php

```diff
@@ -106,6 +134,19 @@ protected function getLocale()
         return $app ? $app->getLocale() : null;
     }
 
+    protected function getFallbackLocale()
+    {
+        if ($this->localeGetter) {
+            return ($this->localeGetter)(true);
```

Comment:
Not sure if this is the correct solution. The alternatives are:

- omit custom fallback locale getter functionality entirely;
- create separate $fallbackLocaleGetter property and corresponding setter for it.

Reply:
Those are for test purpose mainly, so it needs a separate property to be able to test a realistic setup.